packages/block-editor/src/components/block-inspector/index.js

```diff
@@ -270,6 +270,7 @@ const BlockInspectorSingleBlock = ( {
 				} }
 			/>
 			<BlockVariationTransforms blockClientId={ clientId } />
+			<InspectorControls.Slot __experimentalGroup="info" />
```

Comment:
I think that would make more sense directly below `BlockCard` component or we could even consider adding this inside `BlockCard`. I think I had seen some use cases for enhancing the `BlockCard` before, for example adding html(links) like here. The way I had seen others doing it was by modifying the block's description with filters and we used to allow that, even if it should be just a string.

Reply:
If all we want is to render some extra text or links, moving the slot above the block variations makes sense to me. 

I was under the impression that we didn't want to update the BlockCard for the moment, and this new slot would be for ["adding something relevant to the block as a whole"](https://github.com/WordPress/gutenberg/pull/45005#issuecomment-1291762267). That "something" sounded like it could possibly be more than a simple link.

With that flexibility, we get the potential for the slot to be abused though.